guides/hacking.md

```diff
@@ -14,39 +14,27 @@
 
 ## Running the project
 
-### Via dotrun
-
-The simplest way to run the Vanilla framework on Linux is with the [dotrun snap](https://github.com/canonical/dotrun/), and using the `dotrun` command to build the project:
+The simplest way to run the Vanilla framework on all operating systems is by [installing Docker](https://docs.docker.com/engine/installation/) (Linux users may need to [add your user to the `docker` group](https://docs.docker.com/engine/installation/linux/linux-postinstall/)) and [dotrun](https://github.com/canonical/dotrun/), and using the `dotrun` command to build or serve the project:
 
 ```bash
+# Build CSS into the ./build/ directory:
+dotrun build
+
 # Build CSS into the ./build/ directory,
 # and start the server:
 dotrun
 
 # Dynamically watch for changes to the
 # Sass files and build automatically:
 dotrun watch
-```
-
-When writing SCSS, it's useful to run both commands in separate terminals so changes are quickly built and ready for the browser to pick up on page reload.
-
-### Via Docker
-
-It can also be run on other operating systems by first [installing Docker](https://docs.docker.com/engine/installation/) (Linux users may need to [add your user to the `docker` group](https://docs.docker.com/engine/installation/linux/linux-postinstall/)), and then using the `./run` script:
 
-```bash
-# Build CSS into the ./build/ directory:
-./run build
-
-# Start the server:
-./run serve
-
-# Dynamically watch for changes to the
-# Sass files and build automatically:
-./run watch
+# See a full list of commands:
+dotrun exec yarn run --non-interactive
```

Comment:
I've suggested this as a feature of dotrun: https://github.com/canonical/dotrun/issues/106.